tuna.go

```diff
@@ -298,8 +304,9 @@ func (c *Common) CreateServerConn(force bool) error {
 			if subscribersCount == 0 {
 				return errors.New("there is no service providers for " + c.Service.Name)
 			}
-			offset := rand.Intn(int(subscribersCount))
-			subscribers, err := c.Wallet.GetSubscribers(topic, offset, 1, true, false)
+
+			offset := rand.Intn(subscribersCount/getSubscribersBatchSize + 1)
+			subscribers, err := c.Wallet.GetSubscribers(topic, offset, offset*getSubscribersBatchSize, true, false)
```

Comment:
Should be `(topic, offset*getSubscribersBatchSize, getSubscribersBatchSize, true, false)`